helpers/makefstab

```diff
@@ -25,6 +25,45 @@ while (@ARGV) {
   }
 }
 
+sub write_unit {
+  my ($opdir, $uname, $secontext, $m_flagsandoptions, $typ, $source, $mnt_target, $fil, $cont) = @_;
+  open(UNIT, ">$opdir/$uname.mount") or die "Couldn't write to $outputdir/$uname.mount\n";
+
+  $secontext = "";
+  if ($m_flagsandoptions =~ s/(context=[^,]*)//) {
+    $secontext = $1;
+  }
+  # tidy up in case we left leading,trailing or paired ,s
+  $m_flagsandoptions =~ s/,,/,/;
+  $m_flagsandoptions =~ s/^,//;
+  $m_flagsandoptions =~ s/,$//;
+
+  print UNIT <<"END" ;
+[Unit]
+Description=Droid mount for $mnt_target
+Before=local-fs.target systemd-modules-load.service
+
+[Mount]
+What=$source
+Where=$mnt_target
+Type=$typ
+Options=$m_flagsandoptions
+# Options had SELinux context option:  $secontext
+
+# Default is 90 which makes mount period too long in case of
+# errors so drop it down a notch.
+TimeoutSec=10
+
+[Install]
+WantedBy=local-fs.target
+
+# From $fil :
+# $cont
+END
+  print "Wrote $uname.unit\n";
+  close UNIT;
+}
+
 for my $file (@files) {
   open(FILE, "<$file") or die "Can't read from $file\n";
```

Comment:
If building for older android or device bases which do not have /vendor/etc/fstab* files the script should not die here, it should instead have "or next;" to skip not found files.

Reply:
@mlehtima the FSTAB_FILES glob had a wrong bracket, this should now be fixed.